e2e/workload_id/taskapi_test.go

```diff
@@ -129,3 +132,17 @@ func testTaskAPIWindows(t *testing.T) {
 
 	must.StrHasSuffix(t, `"ok":true}}`, logs)
 }
+
+func testTaskAPINomadCLI(t *testing.T) {
+	cluster3.Establish(t,
+		cluster3.LinuxClients(1),
+	)
+	sub, _ := jobs3.Submit(t,
+		"./input/api-nomad-cli.nomad.hcl",
+		jobs3.WaitComplete("grp"),
+	)
+	logs := sub.TaskLogs("grp", "tsk")
+	test.StrContains(t, logs.Stdout, "unix:/") // from `echo $NOMAD_ADDR`
```

Comment:
This is a raw_exec job so we don't know a priori exactly where the secrets dir is, right?

Reply:
correct. I could try to fetch that information, but the goal with this was more to make sure that it didn't connect to Nomad API via some other address, somehow, rather than assert that the whole string is a specific value.